sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedParquetRecordReader.java

```diff
@@ -130,7 +130,7 @@
   /**
    * The memory mode of the columnarBatch
    */
-  private final MemoryMode MEMORY_MODE;
+  private final MemoryMode memoryMode;
```

Comment:
I guess it's fine since the PR is already open but let's avoid making some cosmetic changes alone in a PR.

Reply:
OK ~